apps/edxapp/vars/all/main.yml

```diff
@@ -69,14 +69,23 @@ edxapp_nginx_lms_port: 8071
 edxapp_nginx_htpasswd_secret_name: "edxapp-htpasswd"
 
 # -- celery/redis
-edxapp_celery_lms_high_priority_queue: "edx.lms.core.high"
-edxapp_celery_lms_default_priority_queue: "edx.lms.core.default"
-edxapp_celery_lms_low_priority_queue: "edx.lms.core.low"
-edxapp_celery_lms_high_mem_queue: "edx.lms.core.high_mem"
+#
+# To avoid deploying to much ressouces when it is not required, all queues
+# default to the same default queue name. In this case, only the
+# DeploymentConfig corresponding to queues with a non default name will be
+# created. In other words, if you want to activate a specific queue, you will
+# need to change its name below.
+#
+# Please note that every queue name should be prefixed by the current deployment
+# stamp to avoid treating tasks unrelated with the currently deployed stack.
+edxapp_celery_lms_high_priority_queue: "edx.lms.core.default-{{ deployment_stamp }}"
+edxapp_celery_lms_default_priority_queue: "edx.lms.core.default-{{ deployment_stamp }}"
+edxapp_celery_lms_low_priority_queue: "edx.lms.core.default-{{ deployment_stamp }}"
+edxapp_celery_lms_high_mem_queue: "edx.lms.core.default-{{ deployment_stamp }}"
 
-edxapp_celery_cms_high_priority_queue: "edx.cms.core.high"
-edxapp_celery_cms_default_priority_queue: "edx.cms.core.default"
-edxapp_celery_cms_low_priority_queue: "edx.cms.core.low"
+edxapp_celery_cms_high_priority_queue: "edx.cms.core.default-{{ deployment_stamp }}"
```

Comment:
after seeing the error in CI, I think it makes more sense and solves the problem if the deployment stamp comes before the type of queue (from the most generic to the most specific):
"edx.cms.core.{{ deployment_stamp }}-default"